src/v/cluster/topic_properties.h

```diff
@@ -271,6 +277,8 @@ struct topic_properties
           iceberg_invalid_record_action,
           iceberg_target_lag_ms,
           min_cleanable_dirty_ratio,
+          min_compaction_lag_ms,
+          max_compaction_lag_ms,
```

Comment:
This function defines the serialization order of the properties, so we'll need these to go at the end of the list